README.md

```diff
@@ -120,9 +120,9 @@ setup:
 openstack service create --name monasca monitoring
 openstack service create --name logs logs
 openstack service create --name logs_v2 logs_v2
-openstack endpoint create monasca public http://192.168.10.6:8070/v2.0
-openstack endpoint create logs public http://192.168.10.6:5607/v3.0
-openstack endpoint create logs_v2 public http://192.168.10.6:5607/v2.0
+openstack endpoint create monasca --publicurl http://192.168.10.6:8070/v2.0 --region <Region_name>
```

Comment:
this does not work for me:

```
$ openstack endpoint create monasca --publicurl http://192.168.10.6:8070/v2.0 --region RegionOne
usage: openstack endpoint create [-h] [-f {json,shell,table,value,yaml}]
                                 [-c COLUMN] [--max-width <integer>]
                                 [--fit-width] [--print-empty] [--noindent]
                                 [--prefix PREFIX] [--region <region-id>]
                                 [--enable | --disable]
                                 <service> <interface> <url>
openstack endpoint create: error: argument <interface>: invalid choice: u'http://192.168.10.6:8070/v2.0' (choose from 'admin', 'public', 'internal')
$ openstack --version
openstack 3.12.0
```

Reply:
Sorry, OpenStack client in my env is little old.